### DISCUSSION TITLE
RTL support

### DISCUSSION THREAD

**Discussion by zandiarash** [bitfoundation/bitplatform#8243] [General]
Hi Bitfoundation Team,

Hope you’re doing great! I’m a big fan of **Bitplatform**. Thanks for all your hard work!

I’ve noticed that **Bitplatform** doesn’t fully support RTL (Right-to-Left) languages. This can be a bit tricky for those of us who use these languages regularly, as the text doesn’t always display correctly.
As you know tons of people use RTL languages like :
**Persian, Kurdish, Hebrew, Urdu, Pashto, Yiddish, Syriac, Mandaic, Arabic.**

It would be amazing if you could add full RTL support to **Bitplatform**. Here are a few areas where this would make a huge difference:

- **UI Adjustments:** Menus and toolbars aligning correctly for RTL languages.
- **Bi-directional** Text Support: Mixing RTL and LTR text seamlessly.

Adding this feature would make **Bitplatform** even more user-friendly and inclusive for a wider audience. Thanks for considering this request!

By the way happy **1K** GITHUB STARS 🥳🥳🥳🥳

**Comment by msynk** [comment]
@zandiarash Thanks for contacting us.
I'm unsure what you need since we don't have a Bitplatform product. `bit platform` is the name of our company 😄. So I guess you need this RTL support in our `BlazorUI` component library which has already been implemented and is ready to use.
you can check our RTL samples in the components demo pages or visit this URL:
[https://todo.bitplatform.dev/?culture=fa-IR](https://todo.bitplatform.dev/?culture=fa-IR)

if your request is not supported yet, please provide more information for us.

**Reply by zandiarash** [reply]
> @zandiarash Thanks for contacting us. I'm unsure what you need since we don't have a Bitplatform product. `bit platform` is the name of our company 😄. So I guess you need this RTL support in our `BlazorUI` component library which has already been implemented and is ready to use. you can check our RTL samples in the components demo pages or visit this URL: https://todo.bitplatform.dev/?culture=fa-IR
> 
> if your request is not supported yet, please provide more information for us.

Oh you are right that was my mistake😄.
Actually it had not been implemented since the last time I've tried `BlazorUI` , Thank you a lote
You are always awesome.😍
